udata/tests/forms/test_tag_field.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from udata.tests import TestCase
+from udata.forms.fields import tags_list, slug
+
+
+class TagFieldTest(TestCase):
```

Comment:
It might be even better with a single case by method, ie. `test_accents_are_stripped`, `test_case_is_lowered`...